packaging/opensuse-42.2/snapd.spec

```diff
@@ -149,6 +148,12 @@ go install -s -v -p 4 -x -tags withtestkeys github.com/snapcore/snapd/cmd/snapd
 %gobuild cmd/snapctl
 %gobuild cmd/snap-update-ns
 
+# Removing static link to libseccomp due to libseccomp.a is not available in opensuse
+# This is not affecting due to seccomp is disabled for opensuse
```

Comment:
I believe this comment is not quite correct. We disable apparmor on suse but seccomp will be used. The comment can be updated to something like: `This is ok because snap-seccomp only requires static linking when it runs from the core-snap via re-exec`.